specification/iotsecurity/resource-manager/Microsoft.IoTSecurity/preview/2021-02-01-preview/examples/DefenderSettings/Put.json

```diff
@@ -23,7 +23,8 @@
           "sentinelWorkspaceResourceIds": [
             "/subscriptions/c4930e90-cd72-4aa5-93e9-2d081d129569/resourceGroups/myRg/providers/Microsoft.OperationalInsights/workspaces/myWorkspace1"
           ],
-          "onboardingKind": "Default"
+          "onboardingKind": "Default",
+          "location": "eastus"
```

Comment:
>location [](http://example.com/codeflow?start=11&length=8)

location is a top level resource property (same as 'type'), see [here](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md) for more information. 
Does this means that you can have two different locations?